.github/workflows/debian.yml

```diff
@@ -0,0 +1,34 @@
+name: build debian
+on:
+  schedule:
+    - cron: "0 0 * * 1"
+  push:
+    branches:
+      - master
+    paths:
```

Comment:
Can we merge the debian and alpine builds into a single workflow that uses a matrix to build both?